### DISCUSSION TITLE
Recently video is moving slightly erractic

### DISCUSSION THREAD

**Discussion by tack32** [smplayer-dev/smplayer#476] [General]
The video is no longer smooth. It like there a slight lag in it. It just recently started doing this. I have Version: 22.2.0 (revision 10060) (64-bit). Recent videos play fine in vlc player so I wonder what might have changed in my smplayer setup? 
The video's that aren't playing right (but have always played fine before are from a Sony Action cam) with these statistics:

Resolution: 1920 x 1080
Aspect ratio: 1.77778
Format: h264
Bitrate: 16076 kbps
Frames per second: 29.970030
Selected codec: h264

I just checked another video I downloaded from the internet and it plays fine with these stats:
Resolution: 1280 x 720
Aspect ratio: 1.77778
Format: h264
Bitrate: 1404 kbps
Frames per second: 56.716461
Selected codec: h264

Thanks

**Comment by smplayer-dev** [comment]
It can be many things. Try enabling hardware decoding or increasing the threads for decoding in preference -> performance. You can also try to change the cache settings.

**Reply by tack32** [reply]
Appreciate the suggestions. I'll try to figure some of them out. Thanks!

**Comment by tack32** [comment]
I increased threads by 3 and it seems to have video going smoothly now. Thanks!
